spark/src/main/scala/geotrellis/spark/clip/ClipToGrid.scala

```diff
@@ -132,73 +140,53 @@ object ClipToGrid {
         def coveredBy(e: Extent) = pg.coveredBy(e.toPolygon.jtsGeom)
       }
 
-    def gcPredicates =
```

Comment:
Probably a leftover from a previous signature that involved a `PreparedGeometry`?

Reply:
Ah I didn't think of that. Want me to switch it back and add a comment? Or how about a `lazy val`?